swagger_to_sdk_config.json

```diff
@@ -17,6 +17,9 @@
       "path": "sh .scripts/automation_generate.sh",
       "stderr": {
         "showInComment": ".*ERROR.*"
+      },
+      "cmdout": {
```

Comment:
The property name should be "stdout".

Reply:
No, checkout https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4084754&view=logs&j=47092895-c7c8-567a-c7a6-8c88f3ec178e&s=eb9754a7-3885-5b5b-bd91-16c95dd7881e&t=d293a8e8-c6af-5091-cdf7-4c66482dd6f2&l=511